apps/api/tests/test_chat_api.py

```diff
@@ -194,3 +194,30 @@ def test_start_chat_session_with_auth(team_with_users, authed_client, experiment
     data = {"chatbot_id": experiment.public_id}
     response = authed_client.post(url, data=data, format="json")
     assert response.status_code == 201
+
+@pytest.mark.django_db
+def test_start_chat_session_with_remote_id_and_name(team_with_users, authed_client, experiment):
+    url = reverse("api:chat:start-session")
+    remote_id = "test-remote-id-123"
+    name = "John Doe"
+    session_state = {"ref": "abc123"}
+
+    data = {
+        "chatbot_id": experiment.public_id,
+        "remote_id": remote_id,
+        "name": name,
+        "session_data": session_state,
+    }
+
+    response = authed_client.post(url, data=data, format="json")
+    assert response.status_code == 201
+    response_json = response.json()
+
+    participant = Participant.objects.get(identifier=remote_id)
+    assert response_json["participant"]["identifier"] == remote_id
```

Comment:
This isn't correct. The `remote_id` should not be used as the participant identifier.

I think this should be:
```suggestion
    participant = Participant.objects.get(identifier=response["participant"]["identifier"])
    assert response_json["participant"]["remote_id"] == remote_id
```

(you can add `remote_id` to the output serializer)